source/service.go

```diff
@@ -365,6 +380,11 @@ func (sc *serviceSource) generateEndpoints(svc *v1.Service, hostname string, pro
 		log.Warn(err)
 	}
 
+	externalIp := getExternalIpFromAnnotations(svc.Annotations)
+	if err != nil {
```

Comment:
Should there be an err coming from `getExternalIpFromAnnotations` or should externalIp be checked instead? (as this is, this is checking the err coming from `getTTLFromAnnotations`)

Reply:
I removed the check, not sure it's necessary.

The only thing I could do would be if its not "true" but it exists I could throw an error. What do you think?